### PR TITLE
OPS: Create and use proxy for gradle repo

### PR DESCRIPTION
**Pull request by sebhrusen**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ import org.apache.tools.ant.taskdefs.condition.Os
 // classpath
 buildscript {
     repositories {
-        maven { url 'https://plugins.gradle.org/m2/' }
+        maven {url "${localNexusLocation}/public" }
```
Comment (michalkurka) [inline build.gradle]: only use Local nexus in CI builds:

eg.:

        if (project.ext.isCi) {
            maven {
                url "${localNexusLocation}/public"
            }

File: gradle.properties [hunk 2]
```diff
@@ -40,7 +40,8 @@ doUploadUBenchResults=false
 #
 # Internal Nexus location
 #
-localNexusLocation=http://nexus.0xdata.loc:8081/nexus/repository
```
Comment (michalkurka) [inline gradle.properties]: this keeps changing back and forth, recently Honza changed it to the full URL, is there a reason not to keep it as-is?

Comment (sebhrusen) [reply gradle.properties]: ok, will revert: thought this was old due to the domain name

File: build.gradle (hunk 1 above)
Comment (sebhrusen) [reply build.gradle]: right, otherwise you can't build when not on VPN... fixing this, thx.

Comment (sebhrusen) [reply build.gradle]: @michalkurka should be fixed now, I ran 2 clean builds locally after cleaning gradle cache: once in CI mode and with VPN, one in dev mode and without VPN.

**Pull request merged**